interface/resources/qml/hifi/avatarapp/AvatarsModel.qml

```diff
@@ -21,17 +21,47 @@ ListModel {
 
         return marketItemUrl;
     }
+	
+    function makeMarketThumbnailUrl(marketId) {
+        var avatarThumbnailUrl = "https://hifi-metaverse.s3-us-west-1.amazonaws.com/marketplace/previews/%marketId%/large/hifi-mp-%marketId%.jpg"
+            .split('%marketId%').join(marketId);
+            
+        return avatarThumbnailUrl;
+    }
+	
+    function trimFileExtension(url) {
+        var trimmedUrl = url.substring(0, (url.indexOf("#") == -1) ? url.length : url.indexOf("#"));
```

Comment:
use strict comparisons change `==` to `===` which strictly checks it for the type as well.